packages/medusa/src/repositories/shipping-profile.ts

```diff
@@ -21,5 +21,24 @@ export const ShippingProfileRepository = dataSource
         return acc
       }, {})
     },
+
+    addProducts: async function (
```

Comment:
todo: Change to follow same function definition format as other repositories
```suggestion
    async addProducts(
```